src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <h1 hidden>Stars</h1>
```

Comment:
```suggestion
```

you don't need it (you even hid it) 😉

Reply:
Ok:)
But earlier Misha (as far as I remember) said that it is essential to keep structure as it is has to be. And unrelevant elements can be just hidden.. Is it true?)

Reply:
you may hide element while developing, but you don't need element that user can't even see in production 😉

Reply:
it is just useless 🙃

Reply:
Clear, thank you ^^)